docs/source/tutorial-beginner.rst

```diff
@@ -27,7 +27,6 @@ The code for the Hello World program looks like this.
 
   Make Cozmo say 'Hello World' in this simple Cozmo SDK example program.
   '''
-
```

Comment:
I think I've found the setting that was removing the whitespace. It's saying in SourceTree it's added that blank line now. Look and see.